### PR TITLE
Moneropedia: edit 'Kovri' entry

### PR DESCRIPTION
**Pull request by erciccione**
See discussion in #910

### CODE REVIEW

**Comment by selsta** [comment]
Not sure about this one, we lose quite a bit of information. Maybe re add the "Kovri will protect you and Monero from:" and rename it to I2P?

**Comment by erciccione** [comment]
But that would not be strictly related to Kovri anymore. Maybe we could put some detailed info in a dedicated i2p Moneropedia entry?

File: _i18n/ar/resources/moneropedia/kovri.md [hunk 1]
```diff
@@ -1,61 +1,11 @@
...
 terms: ["Kovri"]
-summary: "Monero's C++ router implementation of the I2P network"
+summary: "C++ router implementation of the I2P network"
 ---
 
-{% include untranslated.html %}
 ### The Basics
 
-[Kovri](https://gitlab.com/kovri-project/kovri/) is a C++ implementation of the @I2P network. @Kovri is currently in heavy, active development and not yet integrated with Monero. When Kovri is integrated into your Monero @node, your transactions will be more secure than ever before.
+[Kovri](https://gitlab.com/kovri-project/kovri/) is a C++ implementation of the I2P network which was proposed to be integrated to Monero. At the moment the future of this I2P router is unclear and an integration with Monero is not planned.
```
Comment (SamsungGalaxyPlayer) [inline _i18n/ar/resources/moneropedia/kovri.md:9]: integrated in Monero

Comment (SamsungGalaxyPlayer) [inline _i18n/ar/resources/moneropedia/kovri.md:9]: integrated in Monero

Comment (erciccione) [reply _i18n/ar/resources/moneropedia/kovri.md]: Changed

**Comment by selsta** [comment]
> Maybe we could put some detailed info in a dedicated i2p Moneropedia entry?

I would like that.

**Comment by erciccione** [comment]
@selsta I opened #1107

**Comment by erciccione** [comment]
Rebased

**Pull request merged**